docs/run/configure-error-mitigation.mdx

```diff
@@ -471,6 +471,37 @@ job = estimator.run(circuits=[psi1], observables=[H1], parameter_values=[theta1]
 psi1_H1 = job.result()
 ```
 
+<span id="no-error-mitigation"></span>
+## Turn off all error mitigation and error suppression
+
+Many researchers want total control over their experiments and don't want the primitives to transform their circuits. To accomplish this, for EstimatorV2, set `resilience_level = 0` and `optimization_level = 0`. For SamplerV2, only measurement twirling is enabled by default. So one needs to set `options.twirling.enable_measure = False`.
```

Comment:
A couple of suggestions:

1. Our built in error mitigation sounds less useful if many researchers don't want them 😛 
2. Simplify the wording to make it easier for the future if/when we enable more by default


```suggestion
You can turn off all error mitigation and suppression if you are, for example, doing research on your own mitigation techniques. To accomplish this, for EstimatorV2, set `resilience_level = 0` and `optimization_level = 0`. For SamplerV2, turn off measurement twirling.
```